cdispyutils/hmac4/hmac4_auth_generator.py

```diff
@@ -7,8 +7,9 @@
 try:
     from urllib.parse import urlparse, parse_qs, quote, unquote, quote_plus
 except ImportError:
-    from urlparse import urlparse, parse_qs
-    from urllib import quote, unquote, quote_plus
```

Comment:
This was Python 3 compatible, `2to3` is not smart enough.

Reply:
oh, right, i will just remove the `try except` since we don't need to support python 2 anymore